lua/telescope/_extensions/gh_builtin.lua

```diff
@@ -0,0 +1,109 @@
+local actions = require('telescope.actions')
+local finders = require('telescope.finders')
+local make_entry = require('telescope.make_entry')
+local previewers = require('telescope.previewers')
+local pickers = require('telescope.pickers')
+local utils = require('telescope.utils')
+local conf = require('telescope.config').values
+local Job = require('plenary.job')
+local builtin = require('telescope.builtin')
+
+local gh_p= require('telescope._extensions.gh_previewers')
+local gh_a= require('telescope._extensions.gh_actions')
+
+local B={ }
+
+local function parse_opts(opts,target)
+  local query={}
+  local tmp_table={}
+  if target=='issue' then
+    tmp_table = {'author' , 'assigner' , 'mention' , 'label' , 'milestone' , 'state' , 'limit' }
+  elseif target=='pr' then
+    tmp_table = {'assigner' , 'label' , 'state' , 'base' , 'limit' }
+  elseif target == 'gist' then
+    tmp_table = {'public' , 'secret'}
+    if opts.public then opts.public =' ' end
+    if opts.secret then opts.secret =' ' end
+  end
+
+  for _, value in pairs(tmp_table) do
+    if opts[value] then
+      table.insert(query,"--" .. value .. ' ' .. opts[value])
+    end
+  end
+  return table.concat(query," ")
+end
+
+-- b for builtin function
+B.gh_issues = function(opts)
+  opts = opts or {}
+  opts.limit = opts.limit or 100
+  local opts_query = parse_opts(opts , 'issue')
+  local cmd = 'gh issue list '.. opts_query
```

Comment:
You can do `local cmd = { 'gh', 'issue', 'list', opts_query }` So you also dont need to do `vim.split(cmd, ' ')`.
And you might need to `flatten` the `cmd` if `opts_query` is a table